collect-pii-with-elements-react/src/RegistrationForm.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react';
+import { TextElement, useBasisTheory } from '@basis-theory/basis-theory-react';
+
+const inputStyle = {
+  fonts: [
+    'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap',
+  ],
+  base: {
+    color: '#fff',
+    fontWeight: 500,
+    fontFamily: "'Source Sans Pro'",
+    fontSize: '16px',
+    fontSmooth: 'antialiased',
+    '::placeholder': {
+      color: '#6b7294',
+    },
+    ':disabled': {
+      backgroundColor: 'transparent',
+    },
+  },
+  invalid: {
+    color: '#ffc7ee',
+  },
+};
+
+const ssnMask = [
+  /\d/u,
+  /\d/u,
+  /\d/u,
+  '-',
+  /\d/u,
+  /\d/u,
+  '-',
+  /\d/u,
+  /\d/u,
+  /\d/u,
+  /\d/u,
+];
+
+export const RegistrationForm = (): JSX.Element => {
+  const [nameEmpty, setNameEmpty] = useState(true);
+  const [ssnComplete, setSsnComplete] = useState(false);
+  const [isBusy, setIsBusy] = useState(false);
+  const [token, setToken] = useState('');
+  const [isError, setIsError] = useState(false);
+  const { bt } = useBasisTheory();
+
+  const submit = async (): Promise<void> => {
+    setIsBusy(true);
+    setIsError(false);
+
+    if (bt) {
+      try {
+        const response = await bt.tokenize({
+          fullName: bt.getElement('fullName'),
+          ssn: bt.getElement('ssn'),
+        });
```

Comment:
Possible enhancement to this request:
```
const response = await bt.tokenize({
    fullName: {
        type: 'token',
        data: bt.getElement('fullName'),
        privacy: {
            classification: 'pii',
            impactLevel: 'low'
        }
    },
    ssn: {
        type: 'social_security_number',
        data: bt.getElement('ssn')
    },
});
```

Reply:
Thanks. I am going to use the `social_security_number` type.